cves/CVE-2017-14058.yml

```diff
@@ -90,18 +95,12 @@ fixes_vcc_instructions: |
   find interesting in there.
 fixes:
 - commit: 7ec414892ddcad88313848494b6fc5f437c9ca4a
-  note: ''
-- commit: ''
-  note: ''
+  note: there are a couple of commits, but they all do the same thing just to different versions of ffmpeg
```

Comment:
Capitalize the first word of the sentence.

Reply:
Thanks fixed